### PR TITLE
refactor: removing duplicate AccountDetails code

### PR DESCRIPTION
**Pull request by cartcrom**
Simplied logic for displaying ENS name instead of wallet address

### CODE REVIEW

File: src/components/AccountDetails/index.tsx [hunk 1]
```diff
@@ -281,59 +281,21 @@ export default function AccountDetails({
...
-                              <Trans>View on Explorer</Trans>
-                            </span>
-                          </AddressLink>
-                        )}
-                      </div>
-                    </AccountControl>
-                  </>
-                )}
+                <AccountControl>
+                  <div>
```
Comment (vm) [inline src/components/AccountDetails/index.tsx:285]: could we remove this extra div? i know it was there before

File: src/components/AccountDetails/index.tsx [hunk 2]
```diff
@@ -129,7 +129,7 @@ const AccountControl = styled.div`
   }
 `
 
-const AddressLink = styled(ExternalLink)<{ hasENS: boolean; isENS: boolean }>`
+const AddressLink = styled(ExternalLink)`
```
Comment (vm) [inline src/components/AccountDetails/index.tsx:132]: were we not using these params?

**Review by vm — APPROVED** [review]

File: src/components/AccountDetails/index.tsx (hunk 2 above)
Comment (cartcrom) [reply src/components/AccountDetails/index.tsx:132]: nope they were just sitting there

File: src/components/AccountDetails/index.tsx (hunk 1 above)
Comment (cartcrom) [reply src/components/AccountDetails/index.tsx:285]: removing it messes up the spacing for some reason

Comment (cartcrom) [reply src/components/AccountDetails/index.tsx:285]: maybe this code should get style-updated later, there some inline stuff and weird structuring

Comment (vm) [reply src/components/AccountDetails/index.tsx:285]: okay cool, yea feel free later, just wanted to see if it was quick

File: src/components/AccountDetails/index.tsx (hunk 2 above)
Comment (vm) [reply src/components/AccountDetails/index.tsx:132]: weird

**Pull request merged**